src/hotspot/share/opto/superword.cpp

```diff
@@ -3382,7 +3434,7 @@ void SuperWord::remove_pack_at(int pos) {
     Node* s = p->at(i);
     set_my_pack(s, nullptr);
   }
-  _packset.remove_at(pos);
+  _packset.at_put(pos, nullptr);
```

Comment:
Note: shifting happens at all call-sites. I just set `nullptr` to be sure the pack does not survive on the packset.